core/assets/css/custom.css

```diff
@@ -342,3 +342,6 @@ form.install > .input-group {
   padding-right: 32px;
   padding-top: 12px;
 }
+.input-group-prepend > i {
+  padding-top: 1.35vh;
```

Comment:
It looks like this should use px/em not vh, but I may be wrong. Did you check how it behaves when resizing your browser?

Reply:
Well, I'll check it, but it should be okay, but yeah, will change it to px, it was too late, so I was little bit off :D